addons/planner_project/static/src/js/planner_project.js

```diff
@@ -0,0 +1,125 @@
+                        'input_element_stage_13' : _t('Reason for cancellation has been documented'),
+                    },
+                    'repair_workshop': {
+                        'input_element_stage_0' : _t('Incoming'),
+                        'input_element_stage_1' : _t('In progress'),
+                        'input_element_stage_2' : _t('Wait. Customer'),
+                        'input_element_stage_3' : _t('Wait. Expert'),
+                        'input_element_stage_4' : _t('Done'),
+                        'input_element_stage_5' : _t('Cancelled'),
+                        'input_element_stage_6' : _t(''),
+                        'input_element_stage_7' : _t('New repair added'),
+                        'input_element_stage_8' : _t('Repair has started'),
+                        'input_element_stage_9' : _t('Feedback from customer requested'),
+                        'input_element_stage_10' : _t('Request for parts has been sent'),
+                        'input_element_stage_11' : _t('Repair is completed'),
+                        'input_element_stage_12' : _t('Customer has cancelled repair'),
+                        'input_element_stage_13' : _t(''),
+                    },
+                    'basic_management': {
+                        'input_element_stage_0' : _t('Ideas'),
+                        'input_element_stage_1' : _t('To Do'),
+                        'input_element_stage_2' : _t('Done'),
+                        'input_element_stage_3' : _t('Cancelled'),
+                        'input_element_stage_4' : _t(''),
+                        'input_element_stage_5' : _t(''),
+                        'input_element_stage_6' : _t(''),
+                        'input_element_stage_7' : _t('Idea is fully explained'),
+                        'input_element_stage_8' : _t('Idea has been transformed into concrete actions'),
+                        'input_element_stage_9' : _t('Task is completed'),
+                        'input_element_stage_10' : _t('Reason for cancellation has been documented'),
+                        'input_element_stage_11' : _t(''),
+                        'input_element_stage_12' : _t(''),
+                        'input_element_stage_13' : _t(''),
+                    }
+                }
+                self.$el.on('change', '#input_element_kanban_stage_pipeline', function(ev) {
+                    var option = $(ev.target).find(":selected").val();
+                    if (_.has(stages, option)) {
+                        var values = stages[option];
+                        var keys = _.keys(values);
+                        for(var i=0; i<keys.length; i++) {
+                            $('#' + keys[i]).val(values[keys[i]]);
+                            $('#' + keys[i] + '_user').val(values[keys[i]]);
+                        }
+                    }
+                });
+                $( ".user_project" ).on('change', function() {
```

Comment:
line 108 will only call if select box change not on inputbox change. It is required because if user change any of inputbox value, same is applied on 'responsiblity' menu inputbox.